server/ngwmn/services/ngwmn.py

```diff
@@ -570,6 +582,8 @@ def get_statistics(agency_cd, site_no):
                         month_stats = monthly[str(month_num)]
                         month_stats['month'] = month_abbr
                         stats['monthly'].append(month_stats)
+                for month in stats['monthly']:
+                    replace_null_values(month, substitution)
```

Comment:
there might be a mix of spaces and tabs?

Reply:
I think it is okay. I had to look at the entire file though.

Reply:
thanks for checking, Mary.